File: templates/templates/FSharp.EpoxyHello.Wpf/EpoxyHello/ViewModels/ItemViewModel.fs
////////////////////////////////////////////////////////////////////////////
//
// Epoxy template source code.
// Write your own copyright and note.
// (You can use https://github.com/rubicon-oss/LicenseHeaderManager)
//
////////////////////////////////////////////////////////////////////////////

namespace EpoxyHello.ViewModels

open Epoxy
open System.Windows.Media

[<Sealed; ViewModel>]
type ItemViewModel() =

    member val Title : string = "" with get, set
    member val Image : ImageSource = null with get, set
    member val Score : int = 0 with get, set

File: templates/templates/EpoxyHello.Uwp/EpoxyHello/ViewModels/ItemViewModel.cs
﻿////////////////////////////////////////////////////////////////////////////
//
// Epoxy template source code.
// Write your own copyright and note.
// (You can use https://github.com/rubicon-oss/LicenseHeaderManager)
//
////////////////////////////////////////////////////////////////////////////

#nullable enable

using Epoxy;
using Windows.UI.Xaml.Media;

namespace EpoxyHello.ViewModels
{
    [ViewModel]
    public sealed class ItemViewModel
    {
        public string? Title { get; set; }

        public ImageSource? Image { get; set; }

        public int Score { get; set; }
    }
}
